imap_processing/tests/mag/test_mag_decom.py

```diff
@@ -118,3 +122,28 @@ def test_mag_raw_cdf_generation(cdf_attrs):
 
     input_xarray = load_cdf(output)
     assert input_xarray.attrs.keys() == burst_data.attrs.keys()
+
+
+def test_comparison():
```

Comment:
Would it make sense to also add a test that loads a ccsds file twice so that you know you are actually pruning duplicate packets? I think this is a nice unit test, but might also be nice to add the larger integration to show it is doing what you want and being used.

I pushed an example up here https://github.com/IMAP-Science-Operations-Center/imap_processing/pull/1850